tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -457,6 +457,7 @@ class ScopedFilterDescriptor {
                  << ToString(status);
     }
 
+#if CUDNN_VERSION >= 5000
```

Comment:
otherwise for CUDNNv4 and less  the `format` variable will be defined but not used (see the other block below:
```c++
    status = dynload::cudnnSetFilterNdDescriptor(parent_, handle_, elem_type,
#if CUDNN_VERSION >= 5000
                                                 format,
#endif
                                                 dims.size(), dims.data());
```